pkg/framework/aries/api/protocol.go

```diff
@@ -32,6 +33,7 @@ type Provider interface {
 	DIDWallet() wallet.DIDCreator
 	Signer() wallet.Signer
 	DIDResolver() didresolver.Resolver
+	DIDStore() didstore.Storage
```

Comment:
TODO: the DID provider options should be part of a verifiable data registry (vdr) option.